aries-test-harness/features/data/waci_issue_credential_attachment.json

```diff
@@ -0,0 +1,63 @@
+[
+  {
+    "id": "df3311ef-894d-4092-9227-22f092d19356",
+    "media_type": "application/json",
+    "lastmod_time": "0001-01-01T00:00:00Z",
+    "data": {
+      "json": {
+        "@context": [
+          "https://www.w3.org/2018/credentials/v1",
+          "https://identity.foundation/credential-manifest/fulfillment/v1"
+        ],
+        "credential_fulfillment": {
+          "id": "a30e3b91-fb77-4d22-95fa-871689c322e2",
+          "manifest_id": "dcc75a16-19f5-4273-84ce-4da69ee2b7fe",
+          "descriptor_map": [
+            {
+              "id": "driver_license_output",
+              "format": "ldp_vc",
+              "path": "$.verifiableCredential[0]"
+            }
+          ]
+        },
+        "proof": {
+          "created": "2021-06-07T20:02:44.730614315Z",
+          "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..NVum9BeYkhzwslZXm2cDOveQB9njlrCRSrdMZgwV3zZfLRXmZQ1AXdKLLmo4ClTYXFX_TWNyB8aFt9cN6sSvCg",
+          "proofPurpose": "authentication",
+          "type": "Ed25519Signature2018",
+          "verificationMethod": "did:orb:EiA3Xmv8A8vUH5lRRZeKakd-cjAxGC2A4aoPDjLysjghow#tMIstfHSzXfBUF7O0m2FiBEfTb93_j_4ron47IXPgEo"
+        },
+        "type": [
+          "VerifiablePresentation",
+          "CredentialFulfillment"
+        ],
+        "verifiableCredential": [
+          {
+            "@context": [
+              "https://www.w3.org/2018/credentials/v1"
+            ],
+            "credentialSubject": {
+              "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+              "license": {
+                "dob": "07/13/80",
+                "number": "34DGE352"
+              }
+            },
+            "id": "https://eu.com/claims/DriversLicense",
+            "issuanceDate": "2010-01-01T19:53:24Z",
+            "issuer": "did:foo:123",
+            "proof": {
+              "created": "2021-06-07T20:02:44.730614315Z",
+              "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..NVum9BeYkhzwslZXm2cDOveQB9njlrCRSrdMZgwV3zZfLRXmZQ1AXdKLLmo4ClTYXFX_TWNyB8aFt9cN6sSvCg",
+              "proofPurpose": "assertionMethod",
+              "type": "Ed25519Signature2018",
+              "verificationMethod": "did:orb:EiA3Xmv8A8vUH5lRRZeKakd-cjAxGC2A4aoPDjLysjghow#tMIstfHSzXfBUF7O0m2FiBEfTb93_j_4ron47IXPgEo"
+            },
+            "type": "VerifiableCredential"
```

Comment:
I believe this property must be an array, but perhaps I am mistaken.

As mentioned in the [VC specification](https://www.w3.org/TR/vc-data-model/#types):
> All [credentials](https://www.w3.org/TR/vc-data-model/#dfn-credential), [presentations](https://www.w3.org/TR/vc-data-model/#dfn-presentations), and encapsulated objects MUST specify, or be associated with, additional more narrow [types](https://www.w3.org/TR/vc-data-model/#dfn-type) (like UniversityDegreeCredential, for example) ...

Reply:
I believe the correct value would be `["VerifiableCredential", "EUDriversLicense"]`. I.e. the spec requires the first value to be the more generic `VerifiableCredential` type, and the follow-up values more concrete sub-types.

Perhaps this usage is also okay, not sure if this would cause issues with other existing VC / Agent implementations.